recipe/meta.yaml

```
@@ -1,44 +1,55 @@
 {% set name = "python-barcode" %}
-{% set version = "0.13.1" %}
+{% set version = "0.15.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: fafba4aa24e9d969777be521c294ff18f6c2b36ad63b5fc2f2108d972e23b252
+  sha256: 3b1825fbdb11e597466dff4286b4ea9b1e86a57717b59e563ae679726fc854de
 
 build:
-  noarch: python
+  skip: True                    # [py<38]
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   entry_points:
     - python-barcode = barcode.pybarcode:main
 
 
 requirements:
   host:
-    - python >=3.5
+    - python
     - pip
     - setuptools_scm
+    - wheel
```

```suggestion
    - setuptools >=45
    - setuptools_scm >=6.2
    - wheel
```
`setuptools` missing, plus some pinnings, see https://github.com/WhyNotHugo/python-barcode/blob/v0.15.1/pyproject.toml#L2

---

That's right I missed that, thanks !